.gitignore

```diff
@@ -1 +1,2 @@
 **/*.swp
+tools
```

Comment:
we don't want to check the tools into git? How will we manage them, or back them up, or track changes?

Reply:
No. Specifically not. What we want is to be able to mess with the tool set without having to mess with the images. One of our current pain points has to do with how we're having to re-do the image every time a newer version of a build tool is required, which means that each image is tailored to a specific (set of) version(s) of our build tool(s).

By keeping this separate, our image is simplified significantly, and it gives us the ability to maintain those tools and configure them, etc., without requiring the baking of new build images.

Reply:
How do we manage the tools folder?

Reply:
Also I don't believe the .gitignore keeps files from being added to the image itself, it only keeps files out of git, unless there's some integration in the image build process to make the docker build consider .gitignore... maybe I'm missing something.